### DISCUSSION TITLE
Allow the HTML `<small>` tag

### DISCUSSION THREAD

**Discussion by notriddle** [community/community#42150] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Writing the following GFM results in the `<small>` tag being stripped out.

````markdown
<small>

```
line1
line2
line3
```

</small>
````

The biggest problem with this isn't that it doesn't work. It's that you can accomplish roughly the same thing using the `<sup>` tag instead, as seen here.

<https://github.com/EmbarkStudios/spirt/tree/0.1.0#simple-example-with-non-trivial-control-flow>

By an apparent coincidence, GitHub's stylesheet forces markdown `<pre>` tags to have a normal line height, even though `<sup>` tags themselves have a `line-height: 0`. Because other GFM renderers might not have exactly the same stylesheet, they wind up mangling this same README file, as seen [in this issue](https://github.com/rust-lang/rust/issues/105906).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @notriddle ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
